File: edukate-checker/src/main/java/io/github/sanyavertolet/edukate/checker/EdukateCheckerApplication.java
package io.github.sanyavertolet.edukate.checker;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication(scanBasePackages = {
        "io.github.sanyavertolet.edukate.checker",
        "io.github.sanyavertolet.edukate.common",
        "io.github.sanyavertolet.edukate.storage"
})
@ConfigurationPropertiesScan(basePackages = "io.github.sanyavertolet.edukate.storage.configs")
public class EdukateCheckerApplication {
    public static void main(String[] args) {
        SpringApplication.run(EdukateCheckerApplication.class, args);
    }
}

File: edukate-checker/src/main/kotlin/io/github/sanyavertolet/edukate/checker/EdukateCheckerApplication.kt
package io.github.sanyavertolet.edukate.checker

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication(
    scanBasePackages =
        [
            "io.github.sanyavertolet.edukate.checker",
            "io.github.sanyavertolet.edukate.common",
            "io.github.sanyavertolet.edukate.storage",
        ]
)
@ConfigurationPropertiesScan(basePackages = ["io.github.sanyavertolet.edukate.storage.configs"])
class EdukateCheckerApplication

@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    runApplication<EdukateCheckerApplication>(*args)
}
